mint-client/src/main.rs

```diff
@@ -47,6 +47,7 @@ enum Command {
     /// Withdraw funds from the federation
```

Comment:
Should the docstring reflect the units of the amount?

Reply:
Or should I just call the variable `satoshis`?

Reply:
Even better.

Reply:
Done